.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -50,6 +56,24 @@ If no open issue exists, you can open one here:
 https://github.com/uswds/uswds/issues/new/choose.
 -->
 
+## Related pull requests
+
+_Indicate if there are other pull requests necessary to complete this issue._
+
+
+<!--
```

Comment:
To match formatting with the other sections, we should remove these extra lines before the comment.

```suggestion
<!--
```